themes/default/views/clients/home.blade.php

```diff
@@ -37,39 +37,26 @@ function removeElement(element) {
                             </div>
                         @endif
                             @foreach ($invoices as $invoice)
-                                @php
+                                <div class="hover:bg-secondary-300 transition-all border-b-[1px] border-secondary-200 ease-out hover:ease-in cursor-pointer w-full p-3" onclick="window.location.href='{{ route('clients.invoice.show', $invoice->id) }}'">
+                                    <div class="flex flex-row">
+                                        <div class="flex my-auto text-2xl ms-2 w-1/12 justify-center text-primary-400">
+                                            <i class="ri-bill-fill"></i>
+                                        </div>
+                                        <div class="w-8/12 ml-4 truncate">
+                                            <span class="font-semibold">{{ __('Invoice ID') }}:</span>
+                                            <span class="font-semibold">{{ $invoice->id }}</span>
+                                    @php
                                     $products = $invoice->getItemsWithProducts()->products;
```

Comment:
This variable does not seem to be used anymore.

Reply:
Removed, thanks